VDA-ERC20/contracts/VDA-V1.sol

```diff
@@ -280,4 +279,23 @@ contract VeridaToken is ERC20PausableUpgradeable, OwnableUpgradeable,
 
         isTransferEnabled = true;
     }
+
+    /**
+     * See {OwnableUpgradeable.sol}
+     * @dev give `mint` role to the new owner
+     */
+    function _transferOwnership(address newOwner) internal virtual override {
```

Comment:
Where is this function called?

Reply:
This function is defined in the `openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol`. And it's called when the contract owner transfer or renounce the ownership.
In this function, we're revoking the `mint` & `Add/Revoke minters` permission from old owner and adding these features to the new owner.

Reply:
Got it, thanks